evaluate/eval.py

```diff
@@ -70,11 +70,14 @@ def eval_report(self):
             prec, rec, rep_recall, f1, rep_f1 = self.evaluate_repairs()
             report = "Precision = %.2f, Recall = %.2f, Repairing Recall = %.2f, F1 = %.2f, Repairing F1 = %.2f, Detected Errors = %d, Total Errors = %d, Correct Repairs = %d, Total Repairs = %d, Total Repairs (Grdth present) = %d" % (
                       prec, rec, rep_recall, f1, rep_f1, self.detected_errors, self.total_errors, self.correct_repairs, self.total_repairs, self.total_repairs_grdt)
+            report_list = [prec, rec, rep_recall, f1, rep_f1, self.detected_errors, self.total_errors,
+                           self.correct_repairs, self.total_repairs, self.total_repairs_grdt]
         except Exception as e:
```

Comment:
Can you modify this exception handling to be similar to the new idiom? 
https://github.com/HoloClean/holoclean/pull/23

i.e. do not silence it but log an error message and re-raise the error.

Reply:
raised exception. shall I log info after merging the latest pull request about logging to avoid conflicts

Reply:
Yes that should be fine.

Reply:
FYI the exception PR has been merged into master https://github.com/HoloClean/holoclean/blob/master/evaluate/eval.py#L72-L76. You'll need to rebase this ontop of master.